src/monattack.cpp

```diff
@@ -293,6 +305,9 @@ void mattack::boomer(monster *z, int index)
 
 void mattack::resurrect(monster *z, int index)
 {
+    if( z->friendly ) {
+        return; // TODO: handle friendly monsters
```

Comment:
Friendly necro ought to either rez corpses as friendly or not rez 'em at all.  It's intelligent and should be helpful.